UI.AutomationTests/Admin/Booking/EditMultiDayHearingTests.cs

```diff
@@ -77,10 +79,36 @@ public async Task EditThisAndUpcomingDaysOfMultiDayHearing()
                 bookingDetailsPage = SearchAndViewHearing(bookingDetailPage, hearingDto);
                 bookingDetailsPage.ValidateDetailsPage(hearingDto);
             }
-            
+            await ValidatorEmailNotifications(hearingDto);
             Assert.Pass();
         }
 
+        private async Task ValidatorEmailNotifications(BookingDto hearingDto)
```

Comment:
maybe rename this to `ValidateEmailNotifications`?

Reply:
Oh, whoops. That's a typo! Thanks